sage-ios/SAGE/SAGE/AdminOperations.swift

```diff
@@ -12,11 +12,15 @@ import SwiftKeychainWrapper
 
 class AdminOperations {
     
-    static func loadMentors(completion: (([User]) -> Void), failure: (String) -> Void){
-
+    static func loadMentors(filter filter: [String: AnyObject]? = nil, completion: (([User]) -> Void), failure: (String) -> Void){
+        var params = [String: AnyObject]()
```

Comment:
you can just pass in filter directly to the parameters argument in the manager.GET method